lib/class-wp-rest-menu-items-controller.php

```diff
@@ -0,0 +1,641 @@
+<?php
+
+/**
+ * Class WP_REST_Menu_Items_Controller
+ */
+class WP_REST_Menu_Items_Controller extends WP_REST_Posts_Controller {
+
+	/**
+	 * Get the post, if the ID is valid.
+	 *
+	 *
+	 * @param int $id Supplied ID.
+	 *
+	 * @return WP_Post|WP_Error Post object if ID is valid, WP_Error otherwise.
+	 */
+	protected function get_post( $id ) {
```

Comment:
Maybe make clear this is a nav item post instead of a regular post in the method name?

Reply:
This is override `get_post` in the post controller. Can't really do anything about the name here.